src/renderer/store/transcriptionWords/ops/restoreSection.ts

```diff
@@ -0,0 +1,22 @@
+import { Op } from 'renderer/store/undoStack/helpers';
+import { IndexRange } from 'sharedTypes';
+import {
+  sectionRestored,
+  undoSectionRestored,
+} from 'renderer/store/transcriptionWords/actions';
+import {
+  RestoreSectionPayload,
+  UndoRestoreSectionPayload,
+} from '../opPayloads';
+
+export type RestoreSectionOp = Op<
+  RestoreSectionPayload,
+  UndoRestoreSectionPayload
+>;
+
+export const makeRestoreSection: (ranges: IndexRange[]) => RestoreSectionOp = (
+  ranges
+) => ({
+  do: [sectionRestored(ranges)],
```

Comment:
Can you make it select the restored words once the restore is complete? And likewise, when undoing a restore, just clear the selection

Reply:
Good idea. Done.